test/jdk/java/awt/Choice/ChoicePopupLocation/ChoicePopupLocation.java

```diff
@@ -120,6 +124,11 @@ private static void openPopup(final Choice choice) throws Exception {
         robot.mouseRelease(InputEvent.BUTTON1_DOWN_MASK);
         robot.waitForIdle();
         if (choice.getSelectedIndex() == 0) {
+            GraphicsConfiguration ge = GraphicsEnvironment
+                    .getLocalGraphicsEnvironment().getDefaultScreenDevice()
+                    .getDefaultConfiguration();
+            BufferedImage failImage = robot.createScreenCapture(ge.getBounds());
```

Comment:
The test verifies all GraphicsDevice in the system, so the screenshot should be done on the screen where the popup is currently shown,